exdb/tests.py

```diff
@@ -197,3 +202,42 @@ def test_load(self):
         self.driver.get(CustomRunner.live_server_url)
         self.assertEquals(self.driver.find_element(By.XPATH, '//h1').text, _('Welcome'))
 
+class PendingApprovalQueueBrowserTest(DefaultLiveServerTestCase):
+    def test_load(self):
+        self.driver.get(CustomRunner.live_server_url + '/pending')
+        self.assertEquals(self.driver.find_element(By.XPATH, '//h1').text, _('Experiences Pending Approval'))
+
+
+"""*******************Integration Tests**********************"""
+
+class StandardTestCase(TestCase):
+    def setUp(self):
+        self.test_user = User.objects.create(username="Test User")
+        self.test_user.set_password('a')
+        self.test_user.save()
+        self.test_type = Type.objects.create(name="Test Type")
+        self.test_sub_type = SubType.objects.create(name="Test Sub Type")
+        self.test_org = Organization.objects.create(name="Test Organization")
+
+class PendingApprovalQueueViewTest(StandardTestCase):
+    def setUp(self):
+        StandardTestCase.setUp(self)
```

Comment:
This method definition is unnecessary. If `PendingApprovalQueueViewTest` doesn't define its own `setUp()` method, the `setUp()` of the parent class (`StandardTestCase`) will be called instead due to inheritance.